README.md

```diff
@@ -150,6 +150,23 @@ pass the dictionary to the method as an additional argument.
 adyen.management.account_company_level_api.get_companies(query_parameters=query_parameters)
 ~~~~
 
+### Using Header Parameters
+
+Define a dictionary named containing the headers you want to include in your request.
+
+~~~~ python    
+    header_parameters = {
+        "Var1": "Var2",
+        "Var2": "Var1"
+    }
+~~~~
+
+Pass the dictionary as an additional argument to the method where you make the API call.
+
+~~~~ python  
+    adyen.checkout.payments_api.payments(header_parameters=headers)
```

Comment:
Spotted a typo here. as the array is called `header_parameters`, this line should say `adyen.checkout.payments_api.payments(header_parameters=header_parameters)`, you could also change the array name in line 158 for increased readability.